content/general/use-latest-version-of-everything.md

```diff
@@ -0,0 +1,32 @@
+---
+title: use latest version of everything
+author: 
+  name: Billy Lando
+  url: https://github.com/billyjov
+---
+
+# Problem 
+
+The Angular team and community are continually improving the ecosystem to make it easier to build applications. Both the performance and the compiler (e.g The Ivy Renderer) are constantly being improved for better web applications.   
+
+Angular as a [release schedules](https://angular.io/guide/releases#schedule) major releases every six months. In most cases, this means that the code should be change within our application. The longer it takes to update, the more expensive will be a future update. Deprecations are available in the [Changelog](https://github.com/angular/angular/blob/master/CHANGELOG.md).
+
+# Solution
+
+**Update your application a few weeks after the release**
```

Comment:
Maybe we can remove this. Or we could talk a bit about agile development and say that it's suggested, once a new major release is out, to plan for an update in the next sprint.

Just an idea.